src/extensions/links.ts

```diff
@@ -41,7 +40,14 @@ export const linkConfigs: Array<PluginExtensionAddedLinkConfig<PluginExtensionPa
       'grafana/alerting/alertingrule/queryeditor',
       ASSISTANT_TARGET_V1,
     ],
-    configure: configureDrilldownLink,
+    configure: configureDrilldownLinkForAsync,
+    onClick: async (_, helpers) => {
+      const context = helpers.context;
+      const url = await buildDrilldownUrlAsync(context);
+      if (url) {
+        window.location.href = url;
+      }
+    },
```

Comment:
Cool! I didn't know that `configure` and `onClick` could be used together. TIL :)